input/groups/armor.json

```diff
@@ -24,7 +24,7 @@
         5891,
         5920
       ],
-      "Basic: Apprentice / Rawhide / Chainmail": [
+      "Basic - Apprentice / Rawhide / Chainmail": [
```

Comment:
I feel like [classifier[: subclassifier]] - [name(s)] is far cleaner, since it's always the same scenario. Few examples, using weapons:

* Dungeon: Ascalonian Catacombs - Royal Ascalonian
* World vs. World - Hero
* Legendary Generation 1 - Experiment
* Legendary Generation 3: Facet of Aurene - Dragon

The hyphen is always the separator no matter what's left or right. Can use pretty much any symbol, but the symbol in the middle should always be the same.

Reply:
Sure.